schemas/eq_submission_schema.json

```diff
@@ -16,83 +16,79 @@
     "metadata",
     "response_id",
     "questionnaire_id",
+    "case_type",
+    "region_code",
     "started_at",
+    "case_id",
     "data"
   ],
   "properties": {
     "tx_id": {
       "$id": "#/properties/tx_id",
       "type": "string",
-      "title": "Transaction ID",
-      "description": "Used to trace a transaction through the whole system",
+      "title": "The Tx_id Schema",
       "default": "",
       "examples": [
-        "f81d4fae-7dec-11d0-a765-00a0c91e6bf6"
+        "411e12a2-e57f-4f07-917d-76c93b422460"
       ],
-      "pattern": "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$"
+      "pattern": "^(.*)$"
     },
     "type": {
       "$id": "#/properties/type",
       "type": "string",
-      "title": "Type",
-      "description": "The unique type identifier of this JSON file. Can be \"uk.gov.ons.edc.eq:surveyresponse\" or \"uk.gov.ons.edc.eq:feedback\"",
```

Comment:
Same as above, review these deletions throughout...

Reply:
We've now reverted the changes that we had inadvertently overwrote.